Stack/Stack.cs

```diff
@@ -0,0 +1,31 @@
+﻿public class TStack<T> {
+   /// <summary>Inserts elements into the stack</summary>

```

Comment:
You have used XML comments. So you should enable 'Documentation file' option in Build/Output in the properties of the project as shown below:
![image](https://github.com/karthick-1205/Assignments/assets/75829713/e39d566a-e9cb-472f-bbb9-c2aa2bbab550)


You should use top-level file comments and also the #regions if you are writing a class.